uiowa/uiowa
# [UIDS] UIDS SCSS path updates

Related to https://github.com/uiowa/uids/pull/915. 

Resolves https://github.com/uiowa/uiowa/issues/7379. 

# How to test
```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu  && ddev drush @sandbox.local uli

```

1. Review Percy visual changes (small font changes due to font update to Google Fonts)
2. Review code changes
3. Click through sandbox site a lot and verify components are styled (make sure your browser cache is cleared and shift + return to reload cached pages)
4. Check browser console logs for missing CSS files
5. Compare https://sandbox.prod.drupal.uiowa.edu/node/2251/edit with https://sandbox.uiowa.ddev.site/node/2251/edit?destination=/admin/content and confirm WYSIWYG preview remains the same, test out styles menu, etc. 
6. Compare https://sandbox.uiowa.ddev.site/node/86/layout with https://sandbox.prod.drupal.uiowa.edu/node/86/layout  and confirm WYSIWYG preview remains the same
